src/native/libs/System.Globalization.Native/pal_localeStringData.c

```diff
@@ -19,11 +19,17 @@ Obtains the value of a DecimalFormatSymbols
 static UErrorCode GetLocaleInfoDecimalFormatSymbol(const char* locale,
                                                    UNumberFormatSymbol symbol,
                                                    UChar* value,
-                                                   int32_t valueLength)
+                                                   int32_t valueLength,
+                                                   int32_t* symbolLength)
 {
     UErrorCode status = U_ZERO_ERROR;
     UNumberFormat* pFormat = unum_open(UNUM_DECIMAL, NULL, 0, locale, NULL, &status);
-    unum_getSymbol(pFormat, symbol, value, valueLength, &status);
+
+    int32_t l =  unum_getSymbol(pFormat, symbol, value, valueLength, &status);
+    if (symbolLength != NULL)
+    {
+        *symbolLength = l;
```

Comment:
```suggestion
        *symbolLength = lengthResult;
```

When I first read this code, I think the `l` was a `1`.